Description: Upstream changes introduced in version 4.4-2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 proguard (4.4-2) unstable; urgency=low
 .
   * update to source format 3.0 (quilt)
 .
 The person named in the Author field signed this changelog entry.
Author: Sam Clegg <samo@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ proguard-4.4/proguard
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec java -Xss300000 -jar /usr/share/java/proguard.jar "$@"
--- /dev/null
+++ proguard-4.4/proguardgui
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec java -Xss300000 -cp /usr/share/java/proguard.jar:/usr/share/java/proguardgui.jar proguard.gui.ProGuardGUI "$@"
